### PR TITLE
Update QUEST.tsv [Lines 1500-2000]

### PR DESCRIPTION
**Pull request by ghost**
Resending pull request #655 to make sure the changes show.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1499,193 +1499,193 @@ xQUEST_20150323_001498	Ok, pay only 1,000 silver then.
...
 QUEST_20150323_001500	Then don't. Pft.
 QUEST_20150323_001501	You're a bum.
-QUEST_20150323_001502	Here you go. You're even more pathetic than me…
+xQUEST_20150323_001502	Here. You're more pathetic than I am…
 QUEST_20150323_001503	Ah, what's up? 
 QUEST_20150323_001504	Farming alone keeps me busy.
-QUEST_20150323_001505	I'm not sure yet… About that… Rebellion.
-QUEST_20150323_001506	It's enticing but…
-QUEST_20150323_001507	Do you have a written letter of Horacius? 
+xQUEST_20150323_001505	I'm not sure yet… about that… rebellion.
```
Comment (vtange) [inline QUEST.tsv]: Old version might be better. The person might be talking about the historic Ruklys Rebellion

File: QUEST.tsv [hunk 2]
```diff
@@ -1499,193 +1499,193 @@ xQUEST_20150323_001498	Ok, pay only 1,000 silver then.
...
+xQUEST_20150323_001591	I can still hear it. {nl}The priests' prayers filling the kingdom. {nl}The endless march of pilgrims coming for the grace of the Goddess.
 QUEST_20150323_001592	It was like that until four years ago. {nl}Before the curse surrounded the cathedral on Divine Day…
 QUEST_20150323_001593	Candlestick
 QUEST_20150323_001594	No candlesticks are lit. 
-QUEST_20150323_001595	I buried Dallas. {nl}It hurts very much.
-QUEST_20150323_001596	Monster again… Run away!
-QUEST_20150323_001597	You were running so hard for me. {nl}Now why don't you give me the key, kid? 
-QUEST_20150323_001598	The gap of contamination, once open, will keep expanding. {nl}Revelator, help Vakarine quickly. 
+xQUEST_20150323_001595	I buried Dallas. {nl}His death pains me so much.
+xQUEST_20150323_001596	Another monster! Run!
```
Comment (vtange) [inline QUEST.tsv]: Might be more than one monster. "More Monsters! Run!" might be better

File: QUEST.tsv [hunk 3]
```diff
@@ -1741,19 +1741,19 @@ QUEST_20150401_001740	I just gave it a while ago…
...
 QUEST_20150401_001743	You must have a lot of questions to ask. Go ahead, Revelator. 
-QUEST_20150401_001744	Vakarine is fighting Valtruss in the Demon Prison across the portal. {nl}She won't hold it for long unless someone helps. 
-QUEST_20150401_001745	The Baron's men went in here. Although they did not make it back.{nl}I watched everything while inside the wizard. {nl}Even the screamings of Vakarine and her Cupols.
-QUEST_20150401_001746	Now let's talk about that. {nl}Whether you are willing to save the Goddess or not.
-QUEST_20150401_001747	I hear the demons that crossed the contaminating gap and are gathered in Aushura Chapel. {nl}I wish you luck in the name of the Goddess. 
+xQUEST_20150401_001744	Vakarine is fighting Valtrus in the Demon Prison across the portal. {nl}She won't be able hold it much longer unless someone helps.
+QUEST_20150401_001745	Men under the command of the Baron have been here though they weren't able to make it back. {nl}I watched everything from inside the wizard. {nl}Even the screams of Vakarine and her Cupols.
+QUEST_20150401_001746	Let's talk about whether or not you're willing to save the Goddess.
```
Comment (vtange) [inline QUEST.tsv]: Big meaning change. We could be talking about something else entirely, irrelevant of the player's will to save the Goddess.

File: QUEST.tsv [hunk 4]
```diff
@@ -1741,19 +1741,19 @@ QUEST_20150401_001740	I just gave it a while ago…
...
+xQUEST_20150401_001744	Vakarine is fighting Valtrus in the Demon Prison across the portal. {nl}She won't be able hold it much longer unless someone helps.
+QUEST_20150401_001745	Men under the command of the Baron have been here though they weren't able to make it back. {nl}I watched everything from inside the wizard. {nl}Even the screams of Vakarine and her Cupols.
 QUEST_20150401_001746	Now let's talk about that. {nl}Whether you are willing to save the Goddess or not.
-QUEST_20150401_001747	I hear the demons that crossed the contaminating gap and are gathered in Aushura Chapel. {nl}I wish you luck in the name of the Goddess. 
+QUEST_20150401_001747	I hear the demons that crossed the corrupting breach are gathered in Aushura Chapel. {nl}I wish you luck in the name of the Goddess. 
 QUEST_20150401_001748	Underground Mausoleum Construction Fee
 QUEST_20150401_001749	Those who kept the honor of King Zachariel and his descendants built the King's grave here. 
 QUEST_20150401_001750	[Things about the secret place is written]
-QUEST_20150401_001751	[Details of Shout the name of the King!]{nl}
```
Comment (ttgmichael) [inline QUEST.tsv:1751]: I think it'll be safer to keep the {nl}'s here. We don't really know where this line resides and how it displays itself yet.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**